src/lib_c/x86_64-windows-msvc/c/dbghelp.cr

```diff
@@ -0,0 +1,135 @@
+@[Link("DbgHelp")]
+lib LibC
+  MAX_SYM_NAME = 2000
+
+  SYMOPT_LOAD_LINES           = 0x00000010
+  SYMOPT_FAIL_CRITICAL_ERRORS = 0x00000200
+  SYMOPT_NO_PROMPTS           = 0x00080000
+
+  struct SYMBOL_INFOW
+    sizeOfStruct : DWORD
+    typeIndex : DWORD
+    reserved : DWORD64[2]
+    index : DWORD
+    size : DWORD
+    modBase : DWORD64
+    flags : DWORD
+    value : DWORD64
+    address : DWORD64
+    register : DWORD
+    scope : DWORD
+    tag : DWORD
+    nameLen : DWORD
+    maxNameLen : DWORD
+    name : WCHAR[1] # VLA
```

Comment:
Couldn't you choose to hardcode the size here (and then not need the `malloc` manipulation)? There are never any uses of it that are non-constant
```suggestion
    name : WCHAR[MAX_SYM_NAME]
```

Reply:
This one is a bit tricky. If we do this then in the following line:

```crystal
symbol.value.sizeOfStruct = sizeof(LibC::SYMBOL_INFOW)
```

changing it to `sizeof(LibC::SYMBOL_INFOW) - (LibC::MAX_SYM_NAME - 1) * sizeof(LibC::WCHAR)` will _not_ work, because somehow `SymFromAddrW` requires the exact size of the original struct with padding. So I am keeping this as close to the original type definition as possible.